CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### 1. Enhancements
 
+#### Elixir
+
+  * [Integer] `digits/2` now accepts negative integers
```

Comment:
Thank you for adding to the changelog but you should avoid doing it in future PRs because it is easy to cause conflicts. It is for your own sake. :D